stix_shifter_modules/verify/configuration/lang_en.json

```diff
@@ -0,0 +1,77 @@
+{
+    "connection": {
+        "help": {
+            "label": "Need additional help?",
+            "description": "More details on the data source setting can be found in the specified link"
+        },
+        "type": {
+            "description": "Data source type"
+        },
+        "name": {
+            "label": "Data source name",
+            "description": "Assign a name to identify the connection (minimum of 5 characters)"
+        },
+        "description": {
+            "label": "Data source description",
+            "description": "Assign a description to indicate the purpose of the connection"
+        },
+        "options": {
```

Comment:
all the parameters inside options already exist in common `stix-shifter/stix_shifter_modules/lang_en.json`. only new parameters specific to this module needs to be added here.